SKRecording/VRHand.cs

```diff
@@ -8,16 +8,18 @@ public class VRHand : ModelWrapper
     {
         private Model handModel;
 
-        struct JointInfo
+        class JointInfo
```

Comment:
Change to a class isn't neccesary, should only be used if we have any methods that the structs needs to support which isn't the case from what i can see

Reply:
I can't loop through it like I do if it's a struct, why not use a class? @nschapeler

Reply:
"Value and Reference Types
There is another difference between structs and classes, and this is also the most important to understand.  Structs are value types, while classes are reference types, and the runtime deals with the two in different ways.  When a value-type instance is created, a single space in memory is allocated to store the value.  Primitive types such as int, float, bool and char are also value types, and work in the same way.  When the runtime deals with a value type, it's dealing directly with its underlying data and this can be very efficient, particularly with primitive types."

Reply:
long story short it's more efficient

Reply:
so since the class is a reference type it takes up more space in memory for its pointer and data whereas the value type is just the data?

Reply:
all we do with this struct is foreach loop through it and read set fields

Reply:
so I guess going with a reference type would be slightly less efficient since we always have to go an extra step when operating on fields?